pkgs/development/ruby-modules/bundled-common/default.nix

```diff
@@ -182,7 +183,7 @@ let
             exit 1
           '';
         };
-    });
+    } // passthru);
```

Comment:
This wouldn't add `passthru` to `basicEnvArgs`, instead it would add the contents of `passthru` to it, `inherit passthru` would add `passthru`

There's already a `passthru` defined so do `//` to it